content/blog/azure-native-v3/index.md

```diff
@@ -0,0 +1,602 @@
+---
+title: "Azure Native V3: A Leaner, Faster SDK for Microsoft Azure"
+date: 2025-04-03
+draft: false
+meta_desc: Azure Native V3 delivers 75% smaller SDK size, updated API versions, and flexible options for working with explicit API versions in Microsoft Azure.
+meta_image: meta.png
+authors:
+    - thomas-kappler
+    - meagan-cojocar
+
+tags:
+    - azure-native
+    - azure
+    - features
+    - releases
+
+social:
+    twitter:
+    linkedin: 
+---
+
+Today we're excited to announce the release of Pulumi Azure Native V3, a major update to our Azure Native provider that improves developer experience through significant SDK size reduction, up-to-date default API versions, and a flexible approach to managing explicit API versions.
+
+[Pulumi Azure Native](/registry/packages/azure-native/) is our native provider for Microsoft Azure, offering direct access to the  [Azure Resource Manager API](https://learn.microsoft.com/en-us/rest/api/resources/). Unlike traditional providers that abstract away cloud APIs, Azure Native gives you more coverage of Azure resources, properties, and up-to-date features by directly mapping to Azure's native REST API. This allows you to provision and manage any Azure resource with Pulumi in your favorite programming language while taking advantage of all the latest Azure capabilities as soon as they're released.
+
+These improvements came about through our [RFC process](https://github.com/pulumi/pulumi-azure-native/issues/4004), where we explored various options to balance SDK size with the flexibility needed by advanced users.
+<!--more-->
+
+## A Slimmer, More Efficient SDK
+
+The most significant improvement in Azure Native V3 is the dramatic 75% reduction in SDK size across all languages. By optimizing the SDK to include only the most commonly used API versions, we've solved one of the most frequent pain points reported by our users:
+
+| Language | Before | After | Reduction | Build Time Before | Build Time After |
```

Comment:
Any reason we left off some languages?

Reply:
I just added Go. Java has never had the explicit API versions, though, and stayed about the same.